server/agent/views.py

```diff
@@ -189,7 +191,19 @@ class AvailableAgentsView(APIView):
     """View to get available agents."""
 
     @swagger_auto_schema(
-        operation_description="Get available Agents",
+        operation_description="Get available Agents", responses={200: AvailableAgentsResponseSerializer()}
     )
     def get(self, request):
-        return Response({"choices": settings.AVAILABLE_AGENTS.keys()}, status=status.HTTP_200_OK)
+        choices = [
+            {
+                "name": name,
+                "args": get_model_descriptor_from_class_field(value["agent"], "PROMPT_ARGS_SCHEMA"),
```

Comment:
PROMPT_ARGS_SCHEMA suggests it's a prompt argument, but if I understand correctly, that's a configuration parameter of an agent, is that right?